use-timescale/data-tiering/about-data-tiering.md

```diff
@@ -7,55 +7,82 @@ tags: [storage, data management]
 
 ## Benefits of the object storage tier
 
-The object storage tier is more than an archiving solution:
+The object storage tier is more than an archiving solution. It is also:
 
-*   **Cost effective.** Store high volumes of data cost-efficiently.
+*   **Cost-effective:** store high volumes of data at a lower cost.
     You pay only for what you store, with no extra cost for queries.
 
-*   **Scalable.**  Scale past the restrictions imposed by storage that can be attached
+*   **Scalable:** scale past the restrictions imposed by storage that can be attached
     directly to a Timescale service (currently 16 TB).
 
-*   **Online.**  Your data is always there and can be [queried when needed][querying-tiered-data]. 
+*   **Online:** your data is always there and can be [queried when needed][querying-tiered-data]. 
 
 ## Architecture
 
-The tiered storage backend works by periodically and asynchronously moving older chunks to the object storage tier;
-an object store built on Amazon S3.
-There, it's stored in the Apache Parquet format, which is a compressed
-columnar format well-suited for S3. Data remains accessible both during and after the migration.
+The tiered storage backend works by periodically and asynchronously moving older chunks to the object storage tier. 
+There, it's stored in the Apache Parquet format, which is a compressed columnar format well-suited for S3. Within a Parquet file, a set of rows is grouped together to form a row group. Within a row group, values for a single column across multiple rows are stored together.
 
 By default, tiered data is not included when querying from a Timescale service. 
-However, it is possible to access tiered data by [enabling tiered reads][querying-tiered-data] for a session, query, or even for all sessions.   
+However, you can access tiered data by [enabling tiered reads][querying-tiered-data] for a query, a session, or even for all sessions. After you enable tiered reads, when you run regular SQL queries, a behind-the-scenes process transparently pulls data from wherever it's located: the standard high-performance storage tier, the object storage tier, or both.
 
-With tiered reads enabled, when you run regular SQL queries, a behind-the-scenes process transparently
-pulls data from wherever it's located: the standard high-performance storage tier, the object storage tier, or both.
 Various SQL optimizations limit what needs to be read from S3:
 
-*   Chunk exclusion avoids processing chunks that fall outside the query's time window
-*   The database uses metadata about row groups and columnar offsets, so only
-    part of an object needs to be read from S3
-
-The result is transparent queries across standard PostgreSQL storage and S3
-storage, so your queries fetch the same data as before.
+* Chunk pruning - exclude the chunks that fall outside the query time window.
```

Comment:
Can you put **Chunk pruning**: etc in bold to match the other lists in the page please.